src/mscorlib/src/System/String.Searching.cs

```diff
@@ -96,8 +96,98 @@ public int IndexOfAny(char[] anyOf, int startIndex)
         }
 
         [Pure]
+        public int IndexOfAny(char[] anyOf, int startIndex, int count)
+        {
+            if (anyOf == null)
+            {
+                throw new ArgumentNullException(nameof(anyOf));
+            }
```

Comment:
@stephentoub The only place I could think of that would be using a 1 char array was the Unix path handling code. That looked to be taken care of with the Path internal changes so I didn't bother with it.
But true to performance testing form the 2 char call is actually 10% better if it has another `if` before it. Not so if I change that to a `switch` though which is interesting. Will run a few more tests on the other paths.